src/internal/utils/global-flags.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+export const awsuiGlobalFlagsSymbol = Symbol.for('awsui-global-flags');
+
+interface GlobalFlags {
+  removeHighContrastHeader?: boolean;
+}
+
+interface ExtendedWindow extends Window {
+  [awsuiGlobalFlagsSymbol]?: GlobalFlags;
```

Comment:
We could potentially add something like `top: ExtendedWindow` to this interface, so that we don't have to do the type assertion for `window.top` below

Reply:
I tried that, but it created a recursive structure that caused complications in testing